string request
---
bool success
geometry_msgs/Pose[] all_parts
string[] type